django/db/backends/mysql/features.py

```diff
@@ -63,6 +63,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_stored_generated_columns = True
     supports_virtual_generated_columns = True
 
+    # Indicates if the database backend supports conditional updates in bulk
+    # operations. True for backends that allow syntax like
+    # "INSERT INTO ... ON CONFLICT (...) DO UPDATE SET ... WHERE ...".
+    # Set to True for backends that support this feature.
+
+    supports_conditional_update = False
+
```

Comment:
This default case should be moved to `django/db/backends/base/features.py` so that it is inherited by all backends by default. _(It should be noted that you've explicitly handled this for 3 out of the 4 built-in backends, i.e. you've forgotten Oracle, so moving this will fix that.)_

I'd also suggest calling in `supports_update_conflicts_where` and moving it adjacent to the existing and related `supports_update_conflicts` flag. I'd also simplify the comment to a single sentence, perhaps adjoining to the existing comments for the `supports_update_conflicts*` flags.

This new flag should also be mentioned under the "Database backend API` section under "Backwards incompatible changes in 5.1" in `docs/releases/5.1.txt`. _(See previous release notes for a guide on the format of this note.)_